src/components/sidebar/sidebar.module.css

```diff
@@ -6,3 +6,15 @@
     margin-left: 1em;
   }
 }
+
+.title,
+.heading {
+  color: var(--token-color-foreground-primary);
+  font-size: var(--token-typography-display-100-font-size);
+  font-weight: var(--token-typography-font-weight-semibold);
+  line-height: var(--token-typography-display-100-line-height);
```

Comment:
Non-blocking: This is _super_ nitpicky, but in the Figma file it looks like `body-100` is being used.

I _cannot_ tell at a glance if there's a difference between `body-100` & `display-100` (there doesn't seem to be), and also `display-100` seems more "correct" in way here since these are heading elements, but wanted to flag just in case ☺️

Reply:
☝️ ahah i had the thought about `compose` as well... tbh sometimes the individual tokens feel more readable to me? Like, `compose` feels very opaque...

I wish we had like "go to definition" like we do in TS, but for `compose`'d classNames 🤔

Reply:
That would be great! I wonder if it'd be easy to make a little vscode extension for it.